packages/block-library/src/post-featured-image/edit.js

```diff
@@ -40,14 +42,11 @@ import DimensionControls from './dimension-controls';
 import OverlayControls from './overlay-controls';
 import Overlay from './overlay';
 import { useToolsPanelDropdownMenuProps } from '../utils/hooks';
+import { unlock } from '../lock-unlock';
+import { DEFAULT_MEDIA_SIZE_SLUG } from './constants';
 
 const ALLOWED_MEDIA_TYPES = [ 'image' ];
-
-function getMediaSourceUrlBySizeSlug( media, slug ) {
-	return (
-		media?.media_details?.sizes?.[ slug ]?.source_url || media?.source_url
-	);
-}
```

Comment:
Why remove this helper?

Reply:
Its used only once, so don't see need for separate function.